lib.go

```diff
@@ -41,6 +41,7 @@ type VM struct {
 // `memoryLimit` is the memory limit of each contract execution (in MiB)
 // `printDebug` is a flag to enable/disable printing debug logs from the contract to STDOUT. This should be false in production environments.
 // `cacheSize` sets the size in MiB of an in-memory cache for e.g. module caching. Set to 0 to disable.
+// `deserCost` sets the gas cost of deserializing one kB of data.
```

Comment:
When we have a fraction, we can charge per byte. I think this is simpler to communicate.

Reply:
It's per byte, I just forgot to update the comment. Fixin'